### PR TITLE
Register RandomQuantum [https://github.com/BBN-Q/RandomQuantum.jl.git] 

### PR DESCRIPTION
**Pull request by marcusps**

### CODE REVIEW

File: RandomQuantum/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.3
```
Comment (tkelman) [inline RandomQuantum/versions/0.0.1/requires]: METADATA is frozen for Julia 0.3 now, but 0.4 or higher is acceptable.

**Comment by tkelman** [comment]
This one looks good if you could rebase out the unrelated changes to other packages. Also good to turn Travis CI on here.

**Comment by blakejohnson** [comment]
Build restarted now that QuantumInfo v0.0.2 has been merged.

File: RandomQuantum/versions/0.0.1/requires [hunk 2]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+QuantumInfo 0.0.2
+Compat
```
Comment (tkelman) [inline RandomQuantum/versions/0.0.1/requires]: this should really have a minimum version that implements the features you need

Comment (tkelman) [inline RandomQuantum/versions/0.0.1/requires]: can probably wait to tag until you have some real changes, it's a minor thing that I'm just encouraging people to be more conscious of

**Pull request merged**